### ISSUE TITLE
No Ripple effect in Vuetify 1.3.0 using Electron (vue-cli-plugin-vuetify)

### ISSUE THREAD

**Issue by 4eveRS** [vuetifyjs/vuetify#5340]
### Versions and Environment
Vuetify: 1.3.0
Vue: 2.5.17
Browsers: Electron 2.0.11
OS: Mojave 10.12

### Previously worked in:
Vuetify: 1.2.10
Vue: 2.5.17

### Steps to reproduce

Create a project following the guidelines:

vue create my-app
vue add electron-builder
vue add vuetify
npm run serve:electron

### Expected Behavior

Buttons should have ripple effect

### Actual Behavior

No ripple effect is present, even when I tried to set it using tje component property ripple.

---

### Additional Comments:
I think maybe this is caused by the new tree shake?
I test with other versions 1.2.9, 1.2.8, 1.2.7 and all worked perfectly.

**Comment by taai** [comment]
It is a bug. I'm pretty sure that the cause of that is undefined variable ```$ripple-animation-transition``` used here:
https://github.com/vuetifyjs/vuetify/blob/v1.3.0/src/stylus/components/_sliders.styl#L198

It stays like that in result CSS too:
```css
transition: $ripple-animation-transition;
```

**Comment by KaelWD** [comment]
Looks like that variable was removed by #5059, but is a separate issue. I'll spin up a CLI project soon and have a look.

**Comment by KaelWD** [comment]
I can't reproduce this issue.

**Comment by taai** [comment]
@KaelWD Download and open the compiled ZIP file of the version 1.3.0 ( https://github.com/vuetifyjs/vuetify/releases/tag/v1.3.0 ), open the ```vuetify-v1.3.0.css``` file and in the line 8610 you will see this line:
```css
  transition: $ripple-animation-transition;
```
– this is how you reproduce this issue. 🙃 

For me this issue exists in SASS when trying to include that CSS file – SASS starts complaining about that [used but undefined] variable. CSS is free, in CSS files there should be no dollar signs. 😉

**Comment by KaelWD** [comment]
I agree that line shouldn't be there, but it has nothing to do with buttons. I followed the instructions you provided and could not find the problem you described. Does the issue only occur because SASS can't find the variable, do ripples work correctly if you load the vuetify CSS normally instead of through SASS?

**Comment by 4eveRS** [comment]
@johnleider The bug still persists, but I found the solution to it.

When creating the new animation/transition of the ripple effect, there's a lot of divisions that can lead to decimal values.

These calculations were introduced when the new ripple effect code was implemented.

[Link to the updated ripple effect files](https://github.com/vuetifyjs/vuetify/pull/5059/files)

Using electron at version @2.0.X or @3.0.x, or even utilising an older version of chrome, makes all the divisions lead to a decimal value using "," instead of ".", knowing that, there's is a issue in older versions of chrome, that makes it only reproduce animations/transitions with a decimal value using a "." and not a ",".
 
Example: 
transition: width 0.3s; (works). 
transition: width 0,3s; (fails)

It's a know bug at chrome and was only fixed in chrome v68.0.3409.0 which is not currently supported by any version of Electron.

[Link to the explanation of the bug in vue/chrome/electron](https://github.com/vuejs/vue/issues/4894)

So when dividing a to calculate the radius and all for the new ripple effect, it can lead to numbers with decimals, which lead to the know bug. That's why I was having intermittent ripple effects. Sometimes the calculation works perfectly and sometimes it makes a decimal and doesn't work at all.

So the fix is easy as it only requires a: .replace(',', '.')) in the variables that control the duration on the transition.
